### PR TITLE
#137 Implement Bookmark Modal

### PR DESCRIPTION
**Pull request by omerfarukbasar**

### CODE REVIEW

File: src/accounts/views.py [hunk 1]
```diff
@@ -79,9 +94,20 @@ def profile_view(request):
...
+                "Log": float(item.get("Log")) if item.get("Log") else None,
+                "Ratings": (
+                    "N/A"
+                    if item.get("Ratings") in [0, "0", None]
+                    else str(item.get("Ratings"))
+                ),
+                "RatingCount": str(item.get("rating_count", 0)),
+                "Category": item.get("Category", "N/A"),
+                "MapLink": f"https://www.google.com/maps/dir/?api=1&destination={quote(item.get('Address'))}",
+                "Distance": "N/A",  # Since distance may not be available
```
Comment (KartikeySharma) [inline src/accounts/views.py]: nit: let's remove comments that are self-explanatory

File: src/accounts/templates/profile_base.html [hunk 2]
```diff
@@ -105,6 +121,53 @@ <h2 class="text-xl font-semibold text-blue-700 mb-4">Edit Profile</h2>
...
+            </button>
+        </div>
+        <div class="mt-3">
+            <h2 id="serviceName" class="text-xl font-semibold text-gray-900 mb-4">Service Name</h2>
+            <div id="serviceMap" class="w-full h-64 rounded-lg overflow-hidden mb-6"></div>
+            <div class="space-y-1 mb-4">
+                <p class="text-sm"><strong>Address:</strong> <span id="serviceAddress">123 Main St</span></p>
+                <p class="text-sm"><strong>Type:</strong> <span id="serviceType">Homeless</span></p>
+                <p class="text-sm"><strong>Rating:</strong> <span id="serviceRating">4.5</span></p>
+<!--                <p class="text-sm"><strong>Distance:</strong> <span id="serviceDistance">1.2 miles</span></p>-->
```
Comment (KartikeySharma) [inline src/accounts/templates/profile_base.html]: Not needed

**Review by KartikeySharma — DISMISSED** [review]
Some nitpicks, rest LGTM! Functionality works.

File: src/accounts/views.py (hunk 1 above)
Comment (omerfarukbasar) [reply src/accounts/views.py]: done in most recent push

File: src/accounts/templates/profile_base.html (hunk 2 above)
Comment (omerfarukbasar) [reply src/accounts/templates/profile_base.html]: Addressed in recent push

**Review by shub-garg — APPROVED** [review]
Wroks great!

**Review by KartikeySharma — APPROVED** [review]
LGTM

File: src/accounts/templates/profile_base.html [hunk 3]
```diff
@@ -117,4 +179,307 @@ <h2 class="text-xl font-semibold text-blue-700 mb-4">Edit Profile</h2>
...
+    document.addEventListener('DOMContentLoaded', () => {
+        function formatTimestamp(timestamp) {
+            return new Date(timestamp).toLocaleString();
+        }
+
+        async function fetchAndDisplayReviews(serviceId, page = 1) {
+            try {
+                const response = await fetch(`/home/get_reviews/${serviceId}/?page=${page}`);
+                if (!response.ok) {
+                    throw new Error(`Failed to fetch reviews. Status: ${response.status}`);
```
Comment (ahhcash) [inline src/accounts/templates/profile_base.html]: move to a static file

**Review by ahhcash — CHANGES_REQUESTED** [review]

**Review by ahhcash — APPROVED** [review]

**Pull request merged**